blog/2020-05-11-version-0.5.0-released.md

```diff
@@ -0,0 +1,102 @@
+---
+id: version-0.5.0-released
+title: Version 0.5.0 released
+author: SourceCred
+author_url: https://github.com/sourcecred
+author_image_url: https://avatars3.githubusercontent.com/u/35711667?s=200&v=4
+---
+
+It's been months since the last versioned release, so there's a lot new to
+v0.5.0!
+
+You can find the release [on GitHub][GitHub release] or as a [Docker image].
+More details on how to use them can be found in the [installation guide].
+
+Since this is a large release, this post only touches on several major additions
+here.
+
+### Discourse plugin
+
+As of this release, you can use SourceCred to analyze [Discourse] forums
+(for example, SourceCred's [own forum]). Our experience so far is that it's
+one of the most interesting plugins, and it works well out of the box!
+
+The basic logic of the Discourse plugin is that every topic, post, and like
+on the forums can flow cred, with cred accumulating at posts that received
+lots of likes, or lots of high-cred responses. Cred also flows between posts
+that reference each other. And the users that authored high cred posts wind up
+with plenty of cred themselves.
+
+The plugin doesn't require any special credentials, so you can easily explore
+Discourse forums you're curious about using SourceCred. It can give a
+perspective on which people are adding content most valued by the community.
+The [plugin setup guide][discourse setup] walks through the commands to do
+this.
+
+### Initiatives
+
+[Initiatives] are added to SourceCred by a new plugin of the same name.
+Instructions are in its [setup guide][initiatives setup].
+
+The goal of the initiatives plugin is to allow tracking important work that
+wasn't directly captured on existing platforms like GitHub or Discourse. You
```

Comment:
Assuming it intentionally simplifies, as there's multiple goals.
It contradicts somewhat with the linked document: https://sourcecred.io/docs/concepts/initiatives
But I'm not interested in spending a lot more time on this post, so leaving as a note.

Reply:
Fine either way